project_national/housing_characteristics/Misc Extra Refrigerator.tsv

```diff
@@ -1,9 +1,11 @@
-Dependency=Geometry Building Type RECS	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 6.7	Option=None	sample_count	sample_weight
-Mobile Home	0.002561	0.007348	0.031474	0.041683	0.002561	0.914373	533.000000	6842068.000000
-Multi-Family with 2 - 4 Units	0.001227	0.007565	0.019774	0.031681	0.001227	0.938526	921.000000	8975958.000000
-Multi-Family with 5+ Units	0.000121	0.001675	0.004964	0.020557	0.000121	0.972562	1900.000000	18886024.000000
-Single-Family Attached	0.005188	0.024873	0.046278	0.072301	0.005188	0.846172	887.000000	6698326.500000
-Single-Family Detached	0.014856	0.054054	0.086323	0.155556	0.014856	0.674355	7743.000000	71332376.000000
+Dependency=Geometry Building Type RECS	Option=EF 6.7	Option=EF 10.2	Option=EF 10.5	Option=EF 15.9	Option=EF 17.6	Option=EF 19.9	Option=None	sample_count	sample_weight
```

Comment:
Seems to be an updated file that wasn't pulled over to ResStock. The distributions are different, not just out of order. There is also an added EF 19.9 option that is not in `develop`.

Reply:
@afontani Do we assume that resstock-estimation has the correct version?

Reply:
@joseph-robertson: yes.